dependency-analysis/README.md

```diff
@@ -0,0 +1,58 @@
+# OpenSSF Scorecard Dependency Analysis
+
+This repository contains the source code for the OpenSSF Dependency Analysis project. The aim of the project is to check the security posture of a project's dependencies using the [GitHub Dependency Graph API](https://docs.github.com/en/rest/dependency-graph/dependency-review?apiVersion=2022-11-28#get-a-diff-of-the-dependencies-between-commits) and the [Security Scorecards API](https://api.securityscorecards.dev).
+
+## Usage
+The OpenSSF Dependency Analysis is a GitHub Action that can be easily incorporated into a workflow. 
+The workflow can be triggered on a pull request event. 
+The action will run on the latest commit on the default branch of the repository, and will create a comment on the pull request with the results of the analysis. 
+An example of the comment can be found [here](https://github.com/ossf-tests/vulpy/pull/2#issuecomment-1442310469).
+
+## Prerequisites
+The actions require enabling the [GitHub Dependency Review](https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-dependency-review) for the repository.
+
+### Configuration
+The action can be configured using the following inputs:
+
+- `SCORECARD_CHECKS`: This environment variable takes a file containing a list of checks to run. 
```

Comment:
We can give users the option to exclude certain repositories or checks for a particular repo, which we can make possible with the file option. 

This will also give us the ability to extend the feature in the future. That's the idea.

Reply:
We should have a config file in this case then. Standard way to pass parameters to an Action is not env variables. It's inputs.